scripts/main.js

```diff
@@ -0,0 +1,74 @@
+/*WebStorage is an abstract class
+for extending our local and session
+storage services*/
+
+class WebStorage {
+    constructor() {
+        if (this.constructor === WebStorage) {
+            throw new TypeError("Can not construct abstract class.");
+        }
+    }
+
+    write() {
```

Comment:
If you define this as a abstract method, you assume that you will override it. But there is no waiting behaviour for this method. Where arguments?